Change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    ax[1].set_ylabel("$Pr(y)$")
</pre><p>I think use $p(y)$, idealy you should refer text in textbook to see what notations Dr. murphy is using.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>